cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/protocol/3.0/casServiceValidationSuccess.jsp

```diff
@@ -49,9 +49,6 @@
                         <cas:${fn:escapeXml(attr.key)}>${fn:escapeXml(attrval)}</cas:${fn:escapeXml(attr.key)}>
                     </c:forEach>
                 </c:forEach>
-                <cas:isFromNewLogin>${fn:escapeXml(isFromNewLogin)}</cas:isFromNewLogin>
```

Comment:
To clarify, this is done because authN attributes are bundled with principal attributes. If no principal attributes are available for release, no authN attributes will be released. The code will make them all be in the same "attributes" bundle.